src/native/linux/qgslinuxnative.cpp

```diff
@@ -16,3 +16,22 @@
  ***************************************************************************/
 
 #include "qgslinuxnative.h"
+
+#include <QUrl>
+#include <QString>
+#include <QtDBus/QtDBus>
+
+void QgsLinuxNative::openFileExplorerAndSelectFile( const QString &path )
+{
+  if ( !QDBusConnection::sessionBus().isConnected() )
+  {
+    QgsNative::openFileExplorerAndSelectFile( path );
+  }
+
+  QDBusInterface iface( QStringLiteral( "org.freedesktop.FileManager1" ),
+                        QStringLiteral( "/org/freedesktop/FileManager1" ),
+                        QStringLiteral( "org.freedesktop.FileManager1" ),
+                        QDBusConnection::sessionBus() );
+
+  iface.call( QStringLiteral( "ShowItems" ), QStringList( QUrl::fromLocalFile( path ).toString() ), QStringLiteral( "QGIS" ) );
```

Comment:
I think we need to check the result here - because if there's no file manager installed which respects the ShowItems message, we want to fall back to the QgsNative method. What does firefox do here?